src/Admin/AmpThemes.php

```diff
@@ -249,4 +254,23 @@ public function filter_themes_api( $override, $action, $args ) {
 
 		return $response;
 	}
+
+	/**
+	 * Add theme metadata for AMP compatibility in theme listing page.
+	 *
+	 * @param string[] $theme_meta An array of the theme's metadata.
+	 * @param string   $stylesheet Directory name of the theme.
+	 *
+	 * @return string[] An array of the theme's metadata.
+	 */
+	public function filter_theme_row_meta( $theme_meta, $stylesheet ) {
+
+		$amp_themes = wp_list_pluck( $this->get_themes(), 'slug' );
+
+		if ( ! empty( $stylesheet ) && in_array( $stylesheet, $amp_themes, true ) ) {
```

Comment:
Is `! empty( $stylesheet )` needed at all? It seems that [`WP_Theme`](https://github.com/WordPress/wordpress-develop/blob/5.8.1/src/wp-includes/class-wp-theme.php#L112-L120) will always return a value for the `$stylesheet` property (and it is what we're ultimately receiving in this filter).

Reply:
Right, and even if it is empty then the `in_array( $stylesheet, $amp_themes, true )` check will then fail.
```suggestion
		if ( in_array( $stylesheet, $amp_themes, true ) ) {
```